spec/models/role_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Role, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Can you add some shoulda validations for relationships here?

Reply:
write tests or don't stage if no tests